docs/integration-guide.md

```
@@ -0,0 +1,766 @@
+**Client validation:**
+- For `pre-registered` clients: validates against `WalletConfig.trustedVerifiers`
+- Can be disabled by setting `**validatePreRegisteredVerifier**` to `false` in wallet config
+
+```swift
+let authorizationRequest: AuthorizationRequest = try await openID4VP.authenticateVerifier(
+    urlEncodedAuthorizationRequest: encodedAuthorizationRequest,
+    shouldValidateClient: true
+)
+```
+
+###### Parameters
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h6

(MD001, heading-increment)